pynitrokey/fido2/client.py

```diff
@@ -358,6 +358,7 @@ def cred_mgmt(self, serial, pin):
                     "Your device has been blocked after too many failed unlock attempts, to fix this it "
                     "will have to be reset. (If no pin is set, plugging it in again might fix this warning)"
                 )
+            raise error
```

Comment:
```suggestion
            raise
```
Alternative (can actually decrease readability, right?)